packages/frontend/src/components/table/props/getScalingTableColumnsConfig.tsx

```diff
@@ -614,3 +616,25 @@ export function getScalingLivenessColumnsConfig() {
   ]
   return columns
 }
+
+export function getScalingFinalityColumnsConfig() {
+  const columns: ColumnConfig<ScalingFinalityViewEntry>[] = [
+    ...getProjectWithIndexColumns({ indexAsDefaultSort: true }),
+    {
+      name: 'DA MODE',
+      getValue: (project) =>
+        project.dataAvailabilityMode ?? <span>&mdash;</span>,
+      tooltip:
+        'The type shows whether projects are posting transaction data batches or state diffs to the L1.',
+    },
+    {
+      name: '30-day avg. time to finality',
+      getValue: (project) => (
+        <FinalityDurationCell data={project.timeToFinalize} />
+      ),
+      tooltip:
+        'The average time it would take for an L2 transaction to be finalized on the L1. Please note, this is an approximate estimation and it also does not take into account L1 reorgs.',
```

Comment:
```suggestion
        'The average time it would take for an L2 transaction to be finalized on the L1. Please note, this is an approximate estimation. For simplicity values ignore the overhead time to reach L1 finality after L1 inclusion, which is shared among all projects.',
```